tasks/Recall Images.md

```diff
@@ -0,0 +1,51 @@
+Archives of Clinical Neuropsychology, 20(6), 745-753. https://watermark.silverchair.com/20-6-745.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAp0wggKZBgkqhkiG9w0BBwagggKKMIIChgIBADCCAn8GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMTbqj_52E6TIpiGSOAgEQgIICUFC6mNKm81A34SENZaX8G3Y5j_lwVxdtUyEkAWT2tJ9EnjutE8SigdXsljqTSxgPtUtBg7_KyHx5b_ekoGXDYeNZRyMr3trC8byBJGjWnVsCbmTLvHwFwfIk3PvJ-_3lwPP1D1RBOp0ICeT2WyJ8GPWnBfHC5pT0lq2vrUM1ORFFv0Wekwe-b_c3CvX9a5mnEHuGuzakXsmk_GzDIaJZ6naP8PI9XdCJBhiZIzsU8Bdr5VhfqZryQm5ksXssiU6LiP0RKT_0N3PsTEIfIy_Tz6FbOBBI_akYJgvqOetbRJOpoWAR4xLtho5cO35Wddou89KuuAJRASo-EsSdkcACY8IleY3SrAOJHN09lbrGTak5leE2CxzCsKXCFFSL_plVz4McoufTH5X0YH3HItLKWtKsTEYO9is743DZ9NmUbDMgIA2_kRXJSFEjHm7oflpbRH_51NdE3yaCig7uKwZJ10zntCG53Qdu5QvYv_pfUJWMPmEeghH1zqzTGWdnahIxvPwXdJw8SQN2TRxNRVKciPmGW_OmqV-Y34j15kQU7eM4nBuUlkJiuWfxGmjcKT2JQHQpAFj5xVXd_ZH79NjfRPCVUep6RUib_1WpNqW8KW2kJ3Hq8Xs9bien1xsuM7rgNF79wg8by46JTWzF6cOg0Eote-iNcUR_Hiqm5qwEkOl4UtBxdkBumYElGUZ3hIyqg1c8p3FY0LiqYPt8fhAjMMUoVGDbs_3lQfJ5bphr0vVLlxkcCuRRnbXL_ido5A2wIEB-M5phry34bH9XF27yq9I
+
+Images: Dennett, H. W., McKone, E., Tavashmi, R., Hall, A., Pidcock, M., Edwards, M., & Duchaine, B. (2012). The
+Cambridge Car Memory Test: A task matched in format to the Cambridge Face Memory Test,
+with norms, reliability, sex differences, dissociations from face memory, and expertise effects.
+Behavior Research Methods, 44(2), 587-605. https://link.springer.com/article/10.3758/s13428-011-0160-2
+
+# Stimuli
+## The visual components
+Example of study set up: ![image](https://user-images.githubusercontent.com/78745728/116818954-e8acf000-ab3b-11eb-9d3d-4fdd576fd625.png)
+Stimuli used in individual tests are based on these images: ![image](https://user-images.githubusercontent.com/78745728/116819043-4b05f080-ab3c-11eb-9bd9-f4bdd6a6258b.png)
+
+
+# Procedure
+## Steps
+1. Participants complete phase one of the experiment which includes series of online tests to measure their individual skill at three problem-solving tasks: Memory,
+Optimization, and Shapes
+2. Participants come to the lab and are randomly assigned to groups of three people. Each group completes a collective version of the individual problem-solving tasks: Memory, Optimization and Shapes. Participants visit the lab twice. During each visit, participants work in two separate groups. Over the course of the experiment, participants are allocated to a total of four groups. The average time between the individual assessment and the first lab visit is 11 days. The average time between the first and second lab visit is 11 days. 
+3. Participants start with the inividual tests,participants completed the images test, in which they were given 20 seconds to memorize six target faces (the stimuli come from the Cambridge Face Memory Test. Participants were then presented with 15 sets of three faces and asked to identify target faces. 
```

Comment:
Didn't they also have to memorize words and stories?
Participants had to memorize list of 12 target words over 24 seconds, then were presented with sets of three words and were asked to identify which, if any of the three, were target words. Last, participants completed the
stories test in which they had 40 seconds to read two short paragraphs, of roughly 60 words each. At the end of the memorization period, participants were asked nine multiple-choice questions about the two paragraphs.